README.md

```diff
@@ -917,7 +917,7 @@ Besides libraries, there're [Collection on Codepen](http://codepen.io/collection
 
 * [es6features](https://github.com/lukehoban/es6features) - Overview of ECMAScript 6 features.
 * [es6-features](https://github.com/rse/es6-features) - ECMAScript 6: Feature Overview & Comparison.
-* [es6-cheatsheet](https://github.com/DrkSephy/es6-cheatsheet) - ES2015 [ES6] cheatsheet containing tips, tricks, best practices and code snippets.
+* [es6-cheat sheet](https://github.com/DrkSephy/es6-cheatsheet) - ES2015 [ES6] cheat sheet containing tips, tricks, best practices and code snippets.
```

Comment:
Maybe the first part can stay as is, it's the project name, changing it doesn't make much sense.